integration/sawtooth_integration/docker/xo-smoke.yaml

```diff
@@ -33,7 +35,11 @@ services:
       - ../../../:/project/sawtooth-core
     expose:
       - 40000
-    entrypoint: ./bin/tp_xo_python validator:40000
+    depends_on:
+      - validator
+    entrypoint: "bash -c \"\
+     sleep 1 && \
```

Comment:
From local tests it seems necessary. I'm wondering if it has to do with dns resolution since each thing that resolves a container name into an address needs to be staggered.